ci-operator/step-registry/workers-rhel/ibmcloud-provision/workers-rhel-ibmcloud-provision-commands.sh

```diff
@@ -11,7 +11,12 @@ function ibmcloud_login {
   region="${LEASED_RESOURCE}"
   export region
   echo "Try to login..."
-  "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+  if [ -f "${SHARED_DIR}/ibmcloud-api-key" ]; then
```

Comment:
We are covering ipi with minimal permission, I think we do not need to update this step.